### ISSUE TITLE
On conda, the displayed badge version is outdated.

### ISSUE THREAD

**Issue by akikuno** [badges/shields#10177]
### Are you experiencing an issue with...

shields.io

### 🐞 Description

On PyPI, the correct latest version (v0.4.1) badge is displayed, but on conda, it shows the older version (v0.3.1).

I confirmed that both the PyPI and conda original sites display the latest version (v0.4.1).
PyPI: https://pypi.org/project/wslPath
Conda: https://anaconda.org/conda-forge/wslpath

### 🔗 Link to the badge

pypi: https://img.shields.io/pypi/v/wslPath.svg
conda: https://img.shields.io/conda/v/conda-forge/wslPath

**Comment by chris48s** [comment]
Hi. When we render this badge we call https://api.anaconda.org/package/conda-forge/wslPath and then report the value of the `latest_version` property, which is `0.3.1`.
I don't really know conda well enough to tell you why `0.4.1` is not considered the latest (even though that is also reported as a known version in the API response). I think that is a question you'd need to ask upstream. Thanks

**Comment by akikuno** [comment]
@chris48s
Thank you very much for your guidance. I will contact Anaconda.
I will update here if there is any progress.
Should I close this issue for now? I'll leave it up to you.

**Comment by PyvesB** [comment]
@akikuno  did you hear back from Anaconda? Will close the issue for the time being as there's nothing actionable from the Shields.io side. 😉

**Comment by akikuno** [comment]
@PyvesB 
Yes, I heard back. The issue was caused by an inconsistency in the anaconda.org API: the `latest_version` field was not updated even though the newer version was present in the `versions` list.  
After re-uploading the package on conda-forge, the API started returning the correct latest version, and the problem was resolved.

So there’s nothing actionable on the Shields.io side. Thanks for following up, and closing the issue makes sense 👍
